packages/assets/package.json

```diff
@@ -2,12 +2,15 @@
   "name": "@react-native/assets-registry",
   "version": "0.73.0",
   "description": "Asset support code for React Native.",
+  "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git@github.com:facebook/react-native.git",
+    "url": "https://github.com/facebook/react-native.git",
     "directory": "packages/assets"
   },
-  "license": "MIT",
+  "homepage": "https://github.com/facebook/react-native/tree/HEAD/packages/assets#readme",
```

Comment:
What is the purpose of `#readme` suffix in all these links? Do we want to navigate to README file?

Reply:
Yup, the that's the purpose of "homepage" field in case of packages under monorepo, unless we've specific page for each package on RN/any other website.
PS. If you look closely exact similar links are already used in RN packages.

Reply:
Also, `#...` is just a "fragment" and won't break urls in any case.